onnxscript/ir/_tape.py

```diff
@@ -32,9 +39,18 @@ def _make_node(self, op_type: str, inputs: Sequence[ir.Value], kwargs: dict[str,
             assert isinstance(outputs, int)
             num_outputs = outputs
 
+        metadata_props = {
+            "pkg.onnxscript.rewriter.stacktrace": _get_stacktrace(),
```

Comment:
Should this always be added? Or, would it be better to do it only if the user sets some config/environment variable?

Reply:
I think it's ok as a default? For example torch.export will add metadata as well. If user wants no stack traces they can run a pass, or if there's demand we can add an option to turn it off?

Reply:
But who's using this? What happens when multiple rewrites happen? Trying to understand what exactly the use-case is and what the users want.